### PR TITLE
fix: go fuzzer build

### PR DESCRIPTION
**Pull request by szuecs**
fix: go fuzzer build

test fuzzer run, maybe it works

### CODE REVIEW

File: fuzz/Dockerfile [hunk 1]
```diff
@@ -6,8 +6,13 @@ ENV PATH="/root/go/bin:$PATH"
 
 COPY . .
 
-RUN dnf install -q -y make clang golang && \
+RUN dnf install -q -y make clang tar gzip       && \
```
Comment (AlexanderYastrebov) [inline fuzz/Dockerfile:9]: Great secure base image that only has outdated go runtimes :shrug: 
```
~$ docker run --rm -it amazonlinux:2023 dnf list golang --showduplicates
Amazon Linux 2023 repository                                        4.0 MB/s |  25 MB     00:06    
Last metadata expiration check: 0:00:12 ago on Tue Mar 19 17:36:31 2024.
Available Packages
golang.x86_64                           1.19.3-2.amzn2023.0.2                            amazonlinux
golang.x86_64                           1.19.6-1.amzn2023.0.1                            amazonlinux
golang.x86_64                           1.19.8-1.amzn2023.0.1                            amazonlinux
golang.x86_64                           1.19.9-1.amzn2023.0.1                            amazonlinux
golang.x86_64                           1.20.5-1.amzn2023.0.2                            amazonlinux
golang.x86_64                           1.20.6-1.amzn2023.0.1                            amazonlinux
golang.x86_64                           1.20.7-1.amzn2023.0.1                            amazonlinux
golang.x86_64                           1.20.8-1.amzn2023.0.1                            amazonlinux
golang.x86_64                           1.20.10-1.amzn2023.0.1                           amazonlinux
golang.x86_64                           1.20.12-1.amzn2023.0.1                           amazonlinux
```

Comment (AlexanderYastrebov) [reply fuzz/Dockerfile:9]:
```
~$ docker run --rm -it amazonlinux:2024 dnf list golang --showduplicates
Unable to find image 'amazonlinux:2024' locally
docker: Error response from daemon: manifest for amazonlinux:2024 not found: manifest unknown: manifest unknown.
See 'docker run --help'.

```
:)

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**